code/modules/clothing/under/miscellaneous.dm

```diff
@@ -169,7 +169,7 @@
 	icon_state = "gear_harness"
 	item_state = "gear_harness"
 	can_adjust = TRUE
-	body_parts_covered = CHEST|GROIN
+	body_parts_covered = NONE
```

Comment:
Might wanna do this change in a modular file if possible

Reply:
Pardon my mistake. It should be fixed it now.